examples/stable/elasticsearch/README.md

```diff
@@ -66,11 +62,7 @@ not. Support for creating an ActionSet as part of install is simply for convenie
 This CR can be shared between Kanister-enabled application instances so one option is to
 only create as part of the first instance.
```

Comment:
Lets change this to the following para since we are no longer installing profile as a part of application install:
```
The command will configure a location where artifacts resulting from Kanister
data operations such as backup should go. This is stored as a `profiles.cr.kanister.io`
*CustomResource (CR)* which is then referenced in Kanister ActionSets. Every ActionSet
requires a Profile reference to complete the action. This CR (`profiles.cr.kanister.io`)
can be shared between Kanister-enabled application instances.
```

Reply:
Updated the para as above.